cli/tests/error_009_op_crates_error.js.out

```diff
@@ -0,0 +1,3 @@
+[WILDCARD]error: Uncaught TypeError: Event requires at least 1 argument, but only 0 present[WILDCARD]
+    at new Event (deno:op_crates/web/[WILDCARD])
```

Comment:
I think this seems pretty reasonable and clean.

@bartlomieju please land if you're okay with it this too.

Reply:
Yeah I agree, @nayeemrmn's right about the host part.